src/SFML/Network/Packet.cpp

```diff
@@ -196,7 +200,8 @@ Packet& Packet::operator >>(Int64& data)
     {
         // Since ntohll is not available everywhere, we have to convert
         // to network byte order (big endian) manually
-        const Uint8* bytes = reinterpret_cast<const Uint8*>(&m_data[m_readPos]);
```

Comment:
Is this memcpy (and the one below) necessary? There should never be alignment issues with uint8.

Reply:
They are not. I just thought it's a good idea to be consistent across all reads from `Packet`. Do you think it's better to leave `reinterpret_cast`s here?